vision/getting-started/image_editing.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    SEGMENTATION_CLASS = ""  # @param {type: 'string'}
</pre><p>Is there an enum/class that can be used for this parameter instead of a string?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>No enum available. Putting all the values in a dropdown would get quite messy in the code so I just formatted everything in the table in the Appendix.</p>